### ISSUE TITLE
Implement XML literal support for scanner

### ISSUE THREAD

**Issue by xeno-by** [twitter/rsc#81]
Looks like there's one more features that's missing from our scanner right now: XML literals. It doesn't seem to be super worth it at this point, given the relative rarity in Scala codebases.

**Comment by rorygraves** [comment]
xml literals are also removed from 2.13 I believe?

**Comment by SethTisue** [comment]
> xml literals are also removed from 2.13 I believe?

no. we're removing scala-xml from the compiler bootstrap, but that's not a user-visible change, it only affects compiler developers.

**Comment by SethTisue** [comment]
> but that's not a user-visible change

(well, I should be careful with my wording... it's user-visible in the narrow sense that the scala-compiler JAR will no longer have a scala-xml dependency, which means a few users may need to tweak their builds. but it doesn't affect scala-xml literal support in the language, that remains as before)

**Comment by rorygraves** [comment]
Thanks for the response.

**Comment by xeno-by** [comment]
I'll be adding some support for XML in https://github.com/twitter/rsc/issues/175, so that we at least do not crash on files with XML literals.
